inc/Lazyload/Component.php

```diff
@@ -144,6 +154,44 @@ public function action_enqueue_lazyload_assets() {
 		wp_script_add_data( 'wp-rig-lazy-load-images', 'precache', true );
 	}
 
+	/**
+	 * Prints the lazy-loading script.
+	 *
+	 * If the browser supports native lazy-loading via the 'loading' attribute, values of data attributes on the images
+	 * will be written back into their real attributes. If the browser does not support native lazy-loading, the script
+	 * bundled in the theme will be loaded.
+	 *
+	 * @see https://web.dev/native-lazy-loading
+	 */
+	public function action_print_lazyload_script() {
+		?>
+		<script type="text/javascript">
+			if ( 'loading' in HTMLImageElement.prototype ) {
+				document.addEventListener( 'DOMContentLoaded', function() {
+					var images = [].slice.call( document.querySelectorAll( 'img.lazy' ) );
+					images.forEach( function( img ) {
+						img.src = img.dataset.src;
+						if ( img.dataset.srcset ) {
+							img.srcset = img.dataset.srcset;
+						}
+						if ( img.dataset.sizes ) {
+							img.sizes = img.dataset.sizes;
+						}
+					} );
+				} );
+			} else {
+				( function() {
+					var script = document.createElement( 'script' );
+					script.type = 'text/javascript';
+					script.src = '<?php echo esc_js( get_theme_file_uri( '/assets/js/lazyload.min.js' ) ); ?>';
```

Comment:
This won't have the version number appended from `wp_rig()->get_asset_version( get_theme_file_path( '/assets/js/lazyload.min.js' ) )`, no?

Reply:
Good catch. Not sure that's a problem though. Also some folks remove those suffixes with a filter, so it would be kind of hard to predict whether we should append it or not.

Now that's a fallback especially, it's unlikely this file will see updates.

Reply:
The same filters could apply as when printing the script via WP_Scripts, then you'd know they are the same. Having the same URL would matter for precaching, I believe, unless the `ver` is stripped. I can't recall at the moment.

But anyway, if you could reference the previously-registerd script in WP_Scripts with the script_loader_src filter applied, then this will ensure consistency and it would allow the URL to be rerouted to another location potentially, such as when plugins provide a CDN for assets.

Reply:
Oh, and minor point: better to use `wp_encode_json()` here than use `esc_js()`, I believe

Reply:
These are good suggestions, however I think this solution would add quite a bit of complexity not really worth it. I'll remove the version query string from that script entirely, this way we won't have to deal with it. It is highly unlikely the file will ever change anyway, especially now that it is a fallback.

Reply:
Good point! 👍